peachjam/templates/peachjam/judgment_detail.html

```diff
@@ -130,6 +130,11 @@
             {% trans 'Read full summary' %}
           </summary>
           {% if perms.peachjam.can_view_document_summary %}
+            {% if document.summary_ai_generated %}
+              <p class="mb-2 text-ai">
```

Comment:
Remember that `p` has its own padding in Bootstrap, so prefer to use div where you know exactly what the result will be.

Reply:
noted,